### ISSUE TITLE
Can't get any value from the plugin

### ISSUE THREAD

**Issue by simonczerwinski** [enkelmedia/Umbraco-SeoVisualizer#8]
Hi @enkelmedia,

I really like the plugin functionalities, simple and smooth.

But I can't get this to work. I am using Umbraco 8.5.3. The problem is when I try to get the Title or the Description value form the CMS, I keep getting null. There is no value in the output.. It seems to get stuck somewhere.
 
This is the code I am using to get data from the plugin: 

```
var seoData = Model.Value<SeoVisualizer.SeoValues>("seo");

<title>@seoData.Title</title>
```

Best regards,
Simon

**Comment by enkelmedia** [comment]
Hi @simonczerwinski!

Thanks for reporting!

There was actually a breaking change in Umbraco 8.1 reg. this:

https://github.com/umbraco/Umbraco-CMS/issues/5170

https://github.com/umbraco/Umbraco-CMS/commit/f039b00a44a57b0d32e956852fa878ac59e4badc#diff-b05c4b135103e2c9aa5cb73b62a9cba5

I guess we would just have to update the property value converter, until this has been done I think you could do something like this:

```
var seoValue = JsonConvert.DeserializeObject<SeoValues>(Model.Value<string>("seo"));
<title>@seoValues.Title</title>
```

NOTE: I have not tested this temp-fix =D

**Comment by simonczerwinski** [comment]
Hi @enkelmedia ,

Thank you very much for the fast respons!!

I see, that was unfortunate but understandable! 

I can say that your method worked perfectly! With a little modification and some null checks, it works very smooth!  :-D

Thanks alot for that!!

Best regards,
Simon

**Comment by enkelmedia** [comment]
Great news!

I’ll re-open the issue here to keep track of this and provide a fix for the property value converter in the near future.

**Comment by enkelmedia** [comment]
Hi!

This has been addressed now in version 8.1.0 of the package, 

https://github.com/enkelmedia/Umbraco-SeoVisualizer/tree/8.1.0

Also published to nuget: https://www.nuget.org/packages/UmbracoSeoVisualizer/
